classes/pdf/PDF.php

```diff
@@ -105,7 +111,7 @@ public function __construct($objects, $template, $smarty, $orientation = 'P')
     public function render($display = true)
     {
         $render = false;
-        $this->pdf_renderer->setFontForLang(Context::getContext()->language->iso_code);
+        $this->pdf_renderer->setLanguage($this->language);
```

Comment:
Personnally, I will setRTL & setFontForLang... because it's not the responsability to PDFGenerator to use Language. What do you think @PrestaShop/prestashop-core-developers ?